src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -0,0 +1,1530 @@
+        self.subtask = subtask
+        self.version = version
+        self.converter = Converter()
+        self.prompt_template = prompt_template
+
+    @property
+    @abstractmethod
+    def task(self) -> str:
+        pass
+
+    @classmethod
+    def download_dataset(cls):
+        target_dir = os.path.join(CLEVA_DATA_PATH, "data")
+        ensure_directory_exists(CLEVA_DATA_PATH)
+        ensure_file_downloaded(source_url=CLEVA_DATA_URL, target_path=target_dir, unpack=True, unpack_type="untar")
+
+    def load_dataset(self) -> Dict[str, List[Dict[str, Any]]]:
+        data_dir: str = os.path.join(CLEVA_DATA_PATH, "data", self.version, self.task)
+        if self.subtask:
+            data_dir = os.path.join(data_dir, self.subtask)
+
+        dataset: Dict[str, List[Dict[str, Any]]] = {}
+        for split in self.splits.keys():
+            if os.path.isfile(os.path.join(data_dir, f"{split}.jsonl")):
+                with open(os.path.join(data_dir, f"{split}.jsonl"), "r") as fin:
+                    dataset[split] = []
+                    for line in fin.readlines():
+                        dataset[split].append(json.loads(line))
+            else:
+                hlog(f"CLEVA:{self.version}:{self.task}:{self.subtask} does not have {split} split")
+
+        return dataset
+
+    @staticmethod
+    def load_prompt_templates(task: str, subtask: Optional[str], version: str) -> List[Dict[str, Any]]:
+        prompt_dir: str = os.path.join(CLEVA_DATA_PATH, "data", version, task)
+        if subtask:
+            prompt_dir = os.path.join(prompt_dir, subtask)
+        file_path = os.path.join(prompt_dir, "prompts.json")
+        if os.path.isfile(file_path):
+            with open(file_path, "r") as fin:
+                prompt_templates: List[Dict[str, Any]] = json.load(fin)
+        else:
+            raise ValueError(f"Missing prompt template file at '{file_path}'")
+        return prompt_templates
+
+    def get_instances(self) -> List[Instance]:
+        # Download the raw data
+        dataset = self.load_dataset()
+
+        # Read all the instances
+        instances: List[Instance] = []
+        for split in self.splits:
+            if split in dataset:
+                for row in dataset[split]:
+                    instances.append(self.process_instance(row, self.splits[split]))
+
+        return instances
+
+    def process_instance(self, row: Dict[str, Any], split: str) -> Instance:
+        instance = self.converter.transform(row, self.prompt_template, split)
+        return instance
+
+    @classmethod
+    def get_prompt_setting(
+        cls, task: str, subtask: Optional[str], version: str, prompt_id: int
+    ) -> Tuple[Dict[str, Any], PromptSetting]:
+        prompt_templates = cls.load_prompt_templates(task, subtask, version)
+        if prompt_id >= len(prompt_templates):
+            raise ValueError(
+                f"You want to use {prompt_id + 1}-th prompt template, but there is only"
```

Comment:
I'd suggest changing `{prompt_id + 1}-th prompt template` to `prompt template with prompt_id {prompt_id}` for simplicity.